CHANGELOG.md

```diff
@@ -37,6 +38,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed minimized pipeline base from `default` to `blank`
 - Removed deprecated `--disable_file_logging` CLI flag
 - Improved flexibility of some pipeline options regarding the application of distortion correction transforms
+- Reverted AFNI version from AFNI_22.3.03 to AFNI_21.1.00
```

Comment:
Maybe
```suggestion
- Pinned AFNI to AFNI_21.1.00
```
instead of reverted since what we had before was `@update.afni.binaries stage` which could be any version (depending on when the Docker image was compiled), not necessarily 22.3.03